images-list

```diff
@@ -248,13 +250,15 @@ k8s.gcr.io/prometheus-adapter/prometheus-adapter rancher/mirrored-prometheus-ada
 k8s.gcr.io/sig-storage/csi-attacher rancher/longhornio-csi-attacher v3.2.1
 k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-longhornio-csi-attacher v3.2.1
 k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-sig-storage-csi-attacher v3.2.0
+k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-sig-storage-csi-attacher v3.3.0
 k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v2.3.0
 k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/mirrored-longhornio-csi-node-driver-registrar v2.3.0
 k8s.gcr.io/sig-storage/csi-provisioner rancher/longhornio-csi-provisioner v2.1.2
 k8s.gcr.io/sig-storage/csi-provisioner rancher/mirrored-longhornio-csi-provisioner v2.1.2
 k8s.gcr.io/sig-storage/csi-provisioner rancher/mirrored-sig-storage-csi-provisioner v2.2.0
 k8s.gcr.io/sig-storage/csi-resizer rancher/longhornio-csi-resizer v1.2.0
 k8s.gcr.io/sig-storage/csi-resizer rancher/mirrored-longhornio-csi-resizer v1.2.0
+k8s.gcr.io/sig-storage/csi-resizer rancher/mirrored-longhornio-csi-resizer v1.3.0
```

Comment:
This is a Longhorn image and shouldn't be used in the vSphere charts as it can be confusing for users regardless of the source image being essentially the same. So this image should actually be:
`k8s.gcr.io/sig-storage/csi-resizer rancher/mirrored-sig-storage-csi-resizer v1.3.0`

However, the latest image the chart uses is from Quay (see [here](https://github.com/rancher/charts/blob/b35ff06e9138b1f221f1b51465755af7ac0ba797/charts/rancher-vsphere-csi/rancher-vsphere-csi/100.1.0%2Bup2.3.0/values.yaml#L33), and [here](https://github.com/rancher/image-mirror/blob/6fb5954a66ba9fe4d3d9230efc4fc55706e86df2/images-list#L541)), so you will need to submit a request to EIO to get the image repository created before it can be mirrored.

Reply:
Sure I will update the PR with these changes.